protobuf/cluster/database.proto

```diff
@@ -23,20 +23,22 @@ option java_outer_classname = "DatabaseProto";
 package grakn.protocol.cluster;
 
 message Database {
-    message Discover {
+
+    message Replica {
+        string address = 1;
+        string database = 2;
+        bool primary = 3;
+        bool preferred_secondary = 4;
+        int64 term = 5;
+    }
```

Comment:
I had actually considered introducing an enum field, `rank`. However, this ended up being unclean, because it's possible for the primary replica to **also** be the preferred secondary replica.